WalletWasabi.Fluent/ViewModels/Wallets/Send/SendViewModel.cs

```diff
@@ -42,6 +40,8 @@ namespace WalletWasabi.Fluent.ViewModels.Wallets.Send
 		NavigationTarget = NavigationTarget.DialogScreen)]
 	public partial class SendViewModel : RoutableViewModel
 	{
+		public bool IsQrButtonVisible { get; }
```

Comment:
We have this `ElementsMustAppearInTheCorrectOrder` rule enabled (which CodeFactor uses, but I don't know why it didn't report it here) https://github.com/zkSNACKs/WalletWasabi/blob/0da3b4e1a90b17933d5a408d6ea1eeeeac550197/Settings.StyleCop#L477

https://documentation.help/StyleCop/SA1201.html

So basically the properties should be placed under the constructor.

Reply:
Got it